src/server.h

```diff
@@ -901,6 +915,20 @@ typedef struct {
                                       need more reserved IDs use UINT64_MAX-1,
                                       -2, ... and so forth. */
 
+/* Replication backlog is not separate memory now, it just is one consumer of
+ * the global replication buffer. This structure records the reference of
+ * replication buffer. Since the replication buffer blocks may be very long,
+ * it would cost much time when we search replication offset on partial resync,
+ * so we use one list to rerord one node every creating 1000 new nodes to make
+ * searching offset from replication buffer blocks list faster. */
```

Comment:
the commit comment and PR can refer to the history.
the code itself should usually document the current state (unless there are backwards compatibility issues, or any other reason to refer to the history)
```suggestion
/* Replication backlog is not separate memory, it just is one consumer of
 * the global replication buffer. This structure records the reference of
 * replication buffers. Since the replication buffer blocks may be very long,
 * it would cost too much to search replication offset on partial resync,
 * so we use one list to rerord one node every 1000 nodes to make
 * searching offset from replication buffer blocks list faster. */
```